### PR TITLE
Update foreman_chef to 0.1.0

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
Scratch build link http://stagingdeb.theforeman.org/pool/plugins/ares/r/ruby-foreman-chef/ruby-foreman-chef_9999-plugin+scratchbuild+201501242045_all.deb 

Build job link http://ci.theforeman.org/view/Packaging/job/packaging_build_deb_plugin/358/

**Comment by mmoll** [comment]
foreman_chef 0.1.0 won't work without foreman_tasks/dynflow, which is not yet packaged for Debian/Ubuntu.

**Comment by ares** [comment]
Yeah, I hope to get foreman-tasks init script fixed soon. It seems we just need to 'debianize' the one we have for rhel?

**Comment by domcleal** [comment]
Looks like you've done a good job on the init script, seems to work fine.

I think there's an error though in foreman_chef 0.1.0 - the initialiser to require dynflow is missing `:before => "foreman_tasks.initialize_dynflow"`.  If I use both this package and #504, plus make that change, the dynflow thingymajig runs successfully.

**Comment by ares** [comment]
Yes, I realized that yesterday. I'll try to release foreman 0.1.1 shortly and I'll update this PR. Thanks

**Comment by ares** [comment]
@domcleal updated to 0.1.1

File: plugins/ruby-foreman-chef/debian/control [hunk 1]
```diff
@@ -8,6 +8,6 @@ Homepage: https://github.com/theforeman/foreman_chef
 
 Package: ruby-foreman-chef
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman
+Depends: ${misc:Depends}, bundler, foreman, ruby-foreman-foreman-tasks (>= 0.6.9), ruby-foreman-foreman-tasks (< 0.7.0)
```
Comment (domcleal) [inline plugins/ruby-foreman-chef/debian/control]: One of the 'foreman's in foreman-tasks' package name has been dropped now

**Comment by ares** [comment]
fixed

**Comment by domcleal** [comment]
Merged as f36fb1f and built into nightlies.  Thanks @ares!

**Pull request closed without merging**